articles/connections/apple-siwa/add-siwa-to-web-app.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Add Sign In with Apple to Web or Other Apps
+description: Learn how to add native login functionality to your web or other app with Apple. 
+toc: true
+topics:
+  - authentication
+  - connections
+  - social
+  - apple
+contentType: how-to
+useCase:
+  - add-login
+  - customize-connections
+  - add-idp
+---
+# Add Sign In with Apple to Web or Other Apps
+
+You can add functionality to your web application that allows your users to use their Apple login credentials to authenticate. As with other identity providers supported by Auth0, when your users login, they can click the "Sign In with Apple" button and they'll be taken to the Apple sign-in screen. They will see the name of your app and a placeholder icon. They will enter their Apple ID and password. If their Apple ID has two-factor authentication enabled, they'll be prompted for that as well.
+
+## Prerequisites
+
+Before you configure Sign In with Apple (SIWA) for your native app in Auth0, do the following:
+
+* Have an [Apple Developer](https://developer.apple.com/programs/) account, which is a paid account with Apple. (There is no free trial available unless you are part of their [iOS Developer University Program](https://developer.apple.com/support/compare-memberships/)).
+
+* A [custom domain](/custom-domains) set up on your Auth0 tenant (because you must be able to do domain verification with Apple).
+
+* [Set Up Apps in the Apple Developer Portal](/connections/apple-siwa/set-up-apple). Make a note of the following IDs and key for the application connection settings in the Auth0 Dashboard:
+
+  * Service ID (Client ID)
+  * Client Secret Signing Key
+  * App ID (Apple Team ID)
```

Comment:
App ID != Apple Team ID

Reply:
I'm not quite sure what you mean with this comment.